### PR TITLE
Discord roundend summary

### PR DESCRIPTION
**Pull request by jupyterkat**
## About The Pull Request

Makes scomo display round end statistics

## Why It's Good For The Game

why not

## Changelog
:cl:
add: Roundend summaries are now on discord
/:cl:

### CODE REVIEW

**Comment by ZombiesVsPlants** [comment]
Based

**Comment by TheFakeElon** [comment]
Couldn't you modularise this? Just comment out the original code if you can't override

**Comment by jupyterkat** [comment]
I’ll make a separate proc for it later

File: austation/code/_HELPERS/roundend.dm [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+		var/discordmsg = ""
+		discordmsg += "--------------ROUND END--------------\n"
+		discordmsg += "Round Number: [GLOB.round_id]\n"
+		discordmsg += "Duration: [DisplayTimeText(world.time - SSticker.round_start_time)]\n"
+		discordmsg += "Players: [GLOB.player_list.len]\n"
+		discordmsg += "Survivors: [survivors]\n"
+		discordmsg += "Escapees: [escapees]\n"
+		discordmsg += "Integrity: [integrity]\n"
+		discordmsg += "Gamemode: [SSticker.mode.name]\n"
+		send2chat(discordmsg, "status")
```
Comment (MCterra10) [inline austation/code/_HELPERS/roundend.dm:12]:
```suggestion
		send2chat(discordmsg, "roundend")
```

File: austation/code/_HELPERS/roundend.dm [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+		discordmsg = ""
+		var/list/ded = SSblackbox.first_death
+		if(ded)
+			discordmsg += "First Death: [ded["name"]], [ded["role"]], at [ded["area"]]\n"
+			var/last_words = ded["last_words"] ? "Their last words were: \"[ded["last_words"]]\"\n" : "They had no last words.\n"
+			discordmsg += "[last_words]\n"
+		else
+			discordmsg += "Nobody died!\n"
+		discordmsg += "--------------------------------------\n"
+		send2chat(discordmsg, "status")
```
Comment (MCterra10) [inline austation/code/_HELPERS/roundend.dm:22]:
```suggestion
		send2chat(discordmsg, "roundend")
```

**Review by MCterra10 — CHANGES_REQUESTED** [review]
I made a dedicated channel for these, change the tags

**Review by MCterra10 — APPROVED** [review]

**Pull request merged**